tensorboard/plugins/profile/tf_profile_dashboard/tf-profile-dashboard.html

```diff
@@ -235,6 +274,17 @@ <h3>No profile data was found.</h3>
         type: Array,
         observer: '_activeHostsChanged'
       },
+      _inColab: {
+        type: Boolean,
+        value: () => !!(window.TENSORBOARD_ENV || {}).IN_COLAB,
```

Comment:
Instead of `!!`, use `Boolean((window.TENSORBOARD_ENV || {}).IN_COLAB)` though I understand both with ease.

Reply:
http://go/tsstyle explicitly says that `!!` is okay; jsstyle has no mention.